final_task/pycalc/expression_calculator.py

```diff
@@ -0,0 +1,109 @@
+from pycalc.constants_and_operations import (FUNCTIONS, OPERATIONS, LOGICAL_OPERATIONS)
+
+
+def has_brackets(expression: list) -> bool:
+    return '(' in expression and ')' in expression
+
+
+def find_brackets(expression: list) -> tuple:
+    end_index = expression.index(')')
+    start_index = end_index - expression[end_index::-1].index('(')
+    return start_index, end_index
+
+
+def remove_brackets(expression: list):
```

Comment:
Here and below:
If one defines function annotations, function output type should be defined as well